### PR TITLE
[API] Add license header to python api client files

### PR DESCRIPTION
**Pull request by Bobgy**
Fixes https://github.com/kubeflow/pipelines/issues/3764

~~Based on https://github.com/kubeflow/pipelines/pull/3895, merge it first~~ UPDATE: #3895 merged
/assign @jingzhang36 @Ark-kun

### CODE REVIEW

File: backend/api/build_kfp_server_api_python_package.sh [hunk 1]
```diff
@@ -56,11 +57,18 @@ java -jar "$codegen_file" generate -g python -i "$swagger_file" -o "$DIR" -c <(e
...
+echo "Copying LICENSE to $DIR"
+cp "$CURRENT_DIR/../../LICENSE" "$DIR"
+
 echo "Building the python package in $DIR."
 pushd "$DIR"
 python3 setup.py --quiet sdist
 popd
 
+echo "Adding license header for generated python files in $DIR."
+go get -u github.com/google/addlicense
```
Comment (Bobgy) [inline backend/api/build_kfp_server_api_python_package.sh:69]: This updates `go.mod` and `go.sum`, is it a good choice?

**Comment by Ark-kun** [comment]
It looks like this PR modifies a lot of files, not just the python client.

**Comment by Bobgy** [comment]
> It looks like this PR modifies a lot of files, not just the python client.

Yes, as stated in description, it is based on #3764, so we should merge it first.
/hold

**Pull request merged**